src/neo/VM/Helper.cs

```diff
@@ -281,45 +281,98 @@ public static byte[] MakeScript(this UInt160 scriptHash, string method, params o
         /// <returns>The <see cref="StackItem"/> represented by a JSON object.</returns>
         public static JObject ToJson(this StackItem item)
         {
-            return ToJson(item, null);
+            int maxSize = int.MaxValue;
```

Comment:
Is the default size too long? 2 billion length string will cost about 4GB memory.

Reply:
Moved it to the parameter.